app/models/item.rb

```diff
@@ -1,3 +1,9 @@
 class Item < ActiveRecord::Base
   belongs_to :list
+  default_scope { order('created_at DESC') }
+
+  def days_left
+    7 - ((DateTime.now.to_date - created_at.to_date).to_i)
+  end
```

Comment:
Would be good to have test coverage in `spec/models/item_spec.rb` for `days_left`.